core/src/config.jl

```diff
@@ -109,10 +109,10 @@ end
 
 # Separate struct, as basin clashes with nodetype
 @option struct Results <: TableOption
-    basin::String = "results/basin.arrow"
-    flow::String = "results/flow.arrow"
-    control::String = "results/control.arrow"
-    allocation::String = "results/allocation.arrow"
+    basin::String = "basin.arrow"
+    flow::String = "flow.arrow"
+    control::String = "control.arrow"
+    allocation::String = "allocation.arrow"
```

Comment:
To fit https://github.com/Deltares/Ribasim/issues/799#issuecomment-1818799618

Those fields need to be removed as far as I can see

Reply:
These fields need to be hardcodes somewhere for core to write them.

Reply:
Sure, but if you hardcode them here, it will be possible to override them with the TOML right?

Reply:
I now always override them.